content/blog/gatsby-multi-author/index.md

```diff
@@ -0,0 +1,231 @@
+- use a yaml transformer because we will store our authors data inside a yaml but you could use JSON or any other format.
+- map Front Matter's `author` attribute to `src/data/author.yaml`.
+- notice that you probably need to `yarn add gatsby-source-filesystem gatsby-transformer-yaml`
+
+```javascript:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    ...otherPlugins,
+
+    // use src/data as a data source
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        path: `${__dirname}/src/data`,
+        name: `data`,
+      },
+    },
+    // enable yaml support
+    `gatsby-transformer-yaml`,
+  ],
+  mapping: {
+    // map author to author.yaml
+    "MarkdownRemark.frontmatter.author": `AuthorYaml`,
+  },
+}
+```
+
+**Important** Front Matter's `author` will need to math an `id` attribute in your `author.yaml`, since that is
+what the mapping uses as a sort of primary key.
+
+Now lets show how the `author.yaml` looks.
+
+```yaml:title=src/data/author.yaml
+- id: author1
+  bio: I am Author 1
+  profilepicture: ../../content/assets/author1.png
+- id: author2
+  bio: I am Author 2
+  profilepicture: ../../content/assets/author2.png
+```
+
+At this point Gatsby will automatically provide the entire Author data via the GraphQL interface,
+in your post template.
+
+```javascript:title=src/templates/blog-post.js
+export const pageQuery = graphql`
+  query BlogPostBySlug($slug: String!) {
+    markdownRemark(fields: { slug: { eq: $slug } }) {
+      excerpt(pruneLength: 160)
+      html
+      frontmatter {
+        title
+        author {
+          id
+          bio
+          profilepicture {
+            childImageSharp {
+              fluid {
+                ...GatsbyImageSharpFluid
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`
+```
+
+Inside your post component you will be able to access all author data:
+
+```javascript:title=src/templates/blog-post.js
+export default function BlogPostTemplate(props) {
+  const {
+    id,
+    bio,
+    profilepicture,
+  } = props.data.markdownRemark.frontmatter.author
+
+  //...
+}
+```
+
+### Bouns: Author profile picture
+
+If you look at the previous snippets you will notice we used the `profilepicture` to
+support author's profile pictures. Notice how in the GraphQL query before we used the
+`childImageSharp` query to enable image optimization, something that the Gatbsy Starter blog
+does by default in the mono-author mode.
+
+By using this `childImageSharp` and any of the other queries of `gatsby-image` you can
+transform dynamically loaded images via the expected grapqhl interface wiht ease.
+
+## How to use all authors
+
+Right now we only cover the pseudo magical way of mapping a Front Matter
+attribute to a filesystem database i.e. `author.yaml`.
+
+But what happens when you want all the authors, lets say for a list of
+authors in your page or a shared about page?
```

Comment:
Maybe link our "about" page as an example? Not sure if it would be worth it but since this post is pretty much the exact implementation we use, maybe the user can see the value directly.

Reply:
done